sympy/core/tests/test_evaluate.py

```diff
@@ -1,9 +1,12 @@
 from sympy.abc import x, y
 from sympy.core.evaluate import evaluate
 from sympy.core import Mul, Add, Pow, S
-from sympy import sqrt
+from sympy import sqrt, oo, pretty
 
 def test_add():
+    with evaluate(False):
+        assert pretty(oo-oo) is not nan
+        
```

Comment:
Maybe you put any extra `tab` characters here. That doesn't pass code quality tests.
I would usually check `test_quality` and the tests related to affected modules on my system before pushing to remote.
E.g, you can do in python console
```py
>>> import sympy
>>> sympy.test('quality', 'core')
```

Reply:
> E.g, you can do in python console
> 
> ```python
> >>> import sympy
> >>> sympy.test('quality', 'core')
> ```

I just did that & both tests passed (from test_evaluate.py file) on my system.

Reply:
> I just did that

In your PR-branch?

I checked there are extra tabs. Just compare line 9 with other empty lines. Remove them, and maybe add more such cases (like `oo -5`, also`oo + 5` and `5+oo` should give the same result).
Did you change similar methods in `NegativeInfinity`? Add test cases with that as well like `-oo + 5`.
I believe then we can merge it.

Reply:
Thanks! I just added more commits.